### ISSUE TITLE
Documentation enhancement

### ISSUE THREAD

**Issue by blackeneth** [JuliaData/DataFrames.jl#2217]

**Comment by bkamins** [comment]
Thank you for the suggestion. I think it is worthwhile to make a PR out of this - would you care to do this. Probably then @nalimilan would be best to review it (he is much better than me in the language-related stuff).

One small note that I think it would be better to take a screenshot of a `DataFrame` output from DataFrames.jl in Julia (so that the users will see here what they will later get when working with the package).

**Comment by timholy** [comment]
Agreed that PRs are best, but discussion in https://discourse.julialang.org/t/non-friendly-documentation/38109/87 is partly about harvesting contributions from users for whom making a PR might be intimidating. The main effort in writing good documentation is the presentation and writing, so we should consider mechanisms to capture that effort.

**Comment by nalimilan** [comment]
A PR can be opened from the GitHub UI, so I don't think it's that scary. And it's also rewarding to be attributed one's work in the commit history.

@blackeneth Feel free to ask for help if you don't find how to do it.

**Comment by blackeneth** [comment]
Ok, I created the pull request. 

I can work on the documentation so more in the near future.

**Comment by bkamins** [comment]
Thank you. This is excellent.

Especially it is great that you are willing to work on enhancing it further. Here, one information is that https://github.com/JuliaData/DataFrames.jl/pull/2214 will change a whole of the documentation related to `select`, `combine`, `by`, `groupby` and `map`, so probably it is best if you skip these parts before that #2214 is merged (hopefully we ca do it in a few days, depending on the availability of the reviewers), as otherwise there will be merge conflicts.
